src/inference/graph_inference.cpp

```diff
@@ -714,12 +775,6 @@ class GraphNodeAssignment : public Nice2Assignment {
   }
 
 private:
-  struct Assignment {
```

Comment:
Because I needed to access the field inside the assignment struct, in order to change the label of a node.